src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+html {
+  --color-border: #e1e7ed;
+}
+
+body {
+  margin: 0;
```

Comment:
```suggestion
```
try not reset margins for body.
it can help to fix tests.
tests expects some spaces horizontally.
![Screenshot from 2023-11-28 14-54-01](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/f2bbe201-b394-4047-9454-25857d6ba4a1)